ext_mongodb.php

```diff
@@ -853,21 +1100,51 @@ public function __debugInfo() : array
 
 final class UTCDateTime implements Type, \Serializable
 {
-	use DenySerialization;
-
 	private int $milliseconds;
 
+	static private function checkArray( array $state )
+	{
+		if (
+			!array_key_exists( 'milliseconds', $state ) ||
+			! ( is_int( $state['milliseconds'] ) || is_string( $state['milliseconds'] ) )
+		) {
+			throw new \MongoDB\Driver\Exception\InvalidArgumentException( "MongoDB\BSON\UTCDateTime initialization requires \"milliseconds\" integer or numeric string field" );
+		}
+	}
+
+	public function serialize() : string
+	{
+		return serialize( [
+			'milliseconds' => (string) $this->milliseconds,
```

Comment:
> as to be the same in each context

HHVM only has one context, so I didn't make the connection.

If you want to use strings here, we should bump PHPC-726 down to 1.2.0 and ensure that we are always serializing the Timestamp and UTCDateTime components as strings. It's necessary for Timestamp since its 32-bit integers are unsigned.

For initializing from a hash, I will continue to support both strings and integers, as that code path is common for my `__set_state()` and unserialization implementations. That not relevant for you, though.